shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/ShivaVariantcalling.scala

```diff
@@ -108,7 +131,9 @@ class ShivaVariantcalling(val parent: Configurable)
               .callersList(this)
               .map(_.name)
               .mkString(", "))
-    if (!callers.exists(_.mergeVcfResults))
+    if (!isGermlineVariantCallingConfigured())
+      Logging.addError("For running the pipeline at least one germline variant caller has to be configured")
```

Comment:
This excludes the possibility run it with only mutect2. It sould be possible to run germline alone, somatic alone and combined. So the only check that should be there is there is at least 1 variant caller, no matter what type.